src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ControllerSecurityConstants.kt

```diff
@@ -40,3 +40,5 @@ const val HAS_VIEW_TIMELINE = """hasAnyAuthority("$EDITOR", "$VIEWER", "$TIMELIN
 
 const val HAS_VIEW_REVIEWS = """hasAnyAuthority("$EDITOR", "$VIEWER", "$REVIEWS_RO", "$REVIEWS_RW")"""
 const val HAS_EDIT_REVIEWS = """hasAnyAuthority("$EDITOR", "$REVIEWS_RW")"""
+
+const val HAS_EDIT_SESSIONS = """hasAnyAuthority("$EDITOR", "$INDUCTIONS_RW", "$REVIEWS_RW")"""
```

Comment:
I did wonder about what roles we'd secure the endpoint with 🤔 
I cant be doing with getting a new role added to the UI client! 🤣